File: 백준/Bronze/10102. 개표/개표.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int V =  Integer.parseInt(sc.nextLine());
        String votes = sc.nextLine(); // 투표 결과 문자열
        
        int countA = 0;
        int countB = 0;
        
        for (int i = 0; i < votes.length(); i++) {
            if (votes.charAt(i) == 'A') {
                countA++;
            } else if (votes.charAt(i) == 'B') {
                countB++;
            }
        }
        
        if (countA > countB) {
            System.out.println('A');
        } else if (countA < countB) {
            System.out.println('B');
        } else {
            System.out.println("Tie");
        }
        
        sc.close();
    }
}

File: 백준/Bronze/10102. 개표/개표.py
v = input()  # 투표 수
votes = input()  # 투표 결과 문자열
aCount = votes.count('A')
bCcount = votes.count('B')

if aCount > bCcount:
    print('A')
elif aCount < bCcount:
    print('B')
else:
    print('Tie')
